doc/getting_started.rst

```diff
@@ -17,29 +17,162 @@
+    few seconds saved add up over the length of the project.
+
+#. Learn the Lingo
+    Video editing is not just a hobby or a profession; it is an industry.  And 
+    just like any industry, there is a ton of jargon to learn.  Practically 
+    speaking, you do not need to know all the terms on 
+    `this list <glossary.html#definitions>`_  to become a better video editor, 
+    but a fundamental knowledge of the terms may help you communicate better 
+    with other video editors or clients.
+
+#. Assemble, Then Make a Rough Cut
+    Drag and drop all your video footage into a timeline and make sure your 
+    frame size and frame rates are consistent.  Begin a new timeline and 
+    drag-and-drop the best clips into what becomes your assembly cut.  Remember 
+    to save your work frequently, and notate the date and time of each version.
+    
+#. Refine Your Video
+    In this phase, your rough cut begins to resemble a cohesive project.
+    Adjust the sound and color, make sure the dialog is audible, and add music,
+    titles, or graphics in this phase.  Color correction is the process of 
+    setting your footage to a color baseline.  No matter how great your subject 
+    looks on set, you will almost always need to do some basic post-processing 
+    for a consistent video.
+ 
+#. Refine Some More
+    A slow scene can set the mood and add tension or it can bore an audience.  
+    A fast scene can add adrenaline to your audience’s systems or it can give 
+    them headaches.  Some editors cut their projects several different ways 
+    before they find the right pace.  Do not let cutting your project several 
+    times discourage you.
+    
+Exporting
+---------
+People view most of their projects on phones, tablets, or computers, so it is 
+essential to know how to export for the web.  The goal when exporting a video 
+for the web is to create the highest quality possible with the smallest file 
+size.  Four main factors determine the file size of your finished video:
+
+* Codec: 
+   A codec determines the type of file format (MP4, AVI, MOV).  The more 
+   compression performed by the codec, the smaller your video’s size.  Videos 
+   that are smaller in file size tend to be lower in visual quality.
+* Resolution: 
+   Resolution refers to the number of horizontal and vertical pixels (dots on 
+   display) your video contains.  For example, a 4K UHD (2160P) video has four 
+   times the resolution of FHD (1080P) video.  A higher resolution means more 
+   information to store so that you will have larger file sizes.
+* Bit Rate: 
+   The Bit Rate is the measure of the speed of data processing of your video.
+   A higher bit rate means higher-quality video and larger files.  OpenShot 
+   allows you to manually set the Bit Rate / Quality in the Advanced tab of the 
+   Export Video window.
+* Frame Rate: 
+   The frequency (in Hz) at which consecutive images, called frames, appear on 
```

Comment:
Somewhere in here we should mention that it's best to editing all the same type of files... for example, consistent with Sample Rate, Channels, Resolution, and Framerate. Source files, Editing Profile, and Export Profile should all match, to reduce the need for mapping and resamplling.